gallery/src/components/NxModal/NxModalPage.tsx

```diff
@@ -28,14 +28,10 @@ export default function NxModalPage() {
   return (
     <>
       <GalleryDescriptionTile>
-        <p className="nx-p">
-          <code className="nx-code">NxModal</code> is the preferred way to handle modals. Invoking
-          an <code className="nx-code">NxModal</code> will create 2 separate <code className="nx-code">div</code>
-          elements. One div will have the <code className="nx-code">nx-modal</code> class, along with any classes passed
-          into the <code className="nx-code">NxModal</code> element with the <code className="nx-code">className</code>
-          attribute. The other div, the parent of the modal, will have
-          the <code className="nx-code">nx-modal-backdrop</code> class applied to it.
```

Comment:
The old text here got too much into implementation details that may continue to evolve over time.